cabot/cabotapp/models.py

```diff
@@ -249,13 +277,20 @@ def update_status(self):
         help_text="URL of service."
     )
 
+    schedule = models.ForeignKey(
```

Comment:
I would recommend a OneToMany mapping here. One service having multiple schedules associated with it.

If a service has an alert, all the on-call schedules should be alerted.